Meta/check-license-headers.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+script_path=$(cd -P -- "$(dirname -- "$0")" && pwd -P)
+cd "$script_path/.."
+
+# We simply check if the file starts with:
+# /*
+#  * Copyright
+PATTERN=$'^/\*\n \* Copyright'
+ERRORS=()
+
+while IFS= read -r f; do
+    if [[ ! -s $f ]]; then
+        continue
+    fi
```

Comment:
Apparently we have empty `.h` files - skip those.

Reply:
Uhh, we can also remove them :D